index.js

```diff
@@ -0,0 +1,114 @@
+class Bank {
+    #accounts
+    #loggedIn
+    #date
+    #transactions
+    constructor(date, accounts) {
+        this.#date = date
+        this.#accounts = accounts
+        this.#loggedIn = false
+    }
+    date(date) {
+        if (typeof date !== "string") {
+            throw "date must be a string"
+        }
+        this.#date = date
+    }
```

Comment:
It doesn't look like this function is ever used outside of tests. Is it necessary? The function name isn't very descriptive, it's not clear what it does from the name alone